src/Message/Command/CommandInterface.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusAlgoliaPlugin\Message\Command;
+
+interface CommandInterface
```

Comment:
The reason for adding this interface is that it makes it easy for plugin users to route all commands to an async transport

Reply:
Ok, small amount of code, but still too early for this imo.

But let's keep it